internal/handlers/pg/pgdb/query_test.go

```diff
@@ -281,3 +282,88 @@ func TestGetDocuments(t *testing.T) {
 		require.NoError(t, err)
 	})
 }
+
+// WHERE clauses occurring frequently in test.
+var (
+	whereEq          string = " WHERE ((_jsonb->$1)::jsonb = $2)"
+	whereEqOrContain string = whereEq + " OR (_jsonb->$1)::jsonb @> $2"
```

Comment:
```suggestion
	idWhereClause          string = " WHERE ((_jsonb->$1)::jsonb = $2)"
	fieldWhereClause string =  " WHERE ((_jsonb->$1)::jsonb = $2) OR (_jsonb->$1)::jsonb @> $2"
```
Just naming suggestion.
It's used in one function, doesn't need to be global var. Let's put them in the `TestPrepareWhereClause` function.